firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -259,3 +272,72 @@ float getInstantaneousSlope(int radius, int xStepsRemaining, int yStepsRemaining
     
     return ((float) radius - xStepsRemaining) / yStepsRemaining;
 }
+
+// Moves upwards to the largest radius in full steps
+void moveToFirstCircle()
+{
+    setScale(yMotor, WHOLE_STEPS);  
+    digitalWrite(yMotor[DIR_PIN], UP);  
+    for (int i = 0; i < circleRadius[0]; i++)
+    {
+        // Y-Step
+        digitalWrite(yMotor[STEP_PIN], LOW);
+        digitalWrite(yMotor[STEP_PIN], HIGH);
+    }
+}
+
+void moveToNextCircle(int currentCircle)
+{
+    // Circle 0 starts at the top, circle 1 at the left, circle 2 at the bottom, continuing in a counterclockwise fashion
+    int quarter = currentCircle % 4;
+
+    // Both motors move in whole steps
+    setScale(xMotor, WHOLE_STEPS);  
+    setScale(yMotor, WHOLE_STEPS);
+
+    // Set Y-direction
+    if (quarter == QUARTER_TOP || quarter == QUARTER_LEFT)
```

Comment:
What are your thoughts on using a ternary operator here? Let me know before you actually implement it, I'd like your thoughts.

Reply:
I just tried it and it looks like a mess lol. Stick with this :)